package.json

```diff
@@ -21,26 +27,24 @@
     "node": ">=10"
   },
   "license": "ISC",
+  "dependencies": {
+    "ansi-escapes": "^4.3.2",
+    "chalk": "^4.1.2",
+    "figures": "^3.2.0",
+    "run-async": "^2.4.1",
+    "rxjs": "^7.5.2"
+  },
   "devDependencies": {
-    "chai": "^4.2.0",
-    "coveralls": "^3.1.0",
+    "c8": "^7.11.0",
+    "chai": "^4.3.4",
     "eslint": "^6.8.0",
-    "flow-bin": "^0.123.0",
+    "flow-bin": "0.123.0",
```

Comment:
I had to pin this because the newer versions result in errors. I'll leave updating this to you later :)

Reply:
I managed to update flow-bin in the last patch, but I'm not familiar with flow. Let me know if this works for you :)